presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClient.java

```diff
@@ -235,6 +237,12 @@ protected String getTableSchemaName(ResultSet resultSet)
         return firstNonNull(resultSet.getString("TABLE_SCHEM"), DEFAULT_SCHEMA);
     }
 
+    @Override
+    public UnsupportedTypeHandling getUnsupportedTypeHandling(ConnectorSession session)
+    {
+        return IGNORE;
```

Comment:
I don't like it. User is allowed to set this catalog session property while it is not used (ignored). I would fix this issue on guice level. To not to use session base jdbc session properties for phoenix at all.

Reply:
Sorry about that. I checked the code and I see that base jdbc session properties are not declared for phoenix. 

I am afraid that jdbc will get more session properties in near future and will hit similar issues like this one.

Reply:
> I am afraid that jdbc will get more session properties in near future and will hit similar issues like this one.

I think so too. Actually, I tried to bind or isolate Phoenix connector from base jdbc, but I couldn't find the better way and settled on this workaround to resolve regression.